src/components/PageHero.tsx

```diff
@@ -157,10 +119,24 @@ const PageHero: React.FC<IProps> = ({
             </Wrap>
           )}
           {children}
-        </HeroContent>
-        <HeroImg image={image} objectFit="contain" alt={alt} loading="eager" />
-      </HeroContainer>
-    </Content>
+        </Box>
+        <Box
+          as={GatsbyImage}
+          flex="1 1 50%"
+          backgroundSize="cover"
+          backgroundRepeat="no-repeat"
+          alignSelf="center"
+          mt={{ base: 0, lg: 12 }}
+          ml={{ base: 0, lg: 12 }}
+          w="full"
+          maxWidth={{ base: "560px", lg: "624px" }}
+          image={image}
+          objectFit="contain"
+          alt={alt}
+          loading="eager"
+        />
```

Comment:
This needs to be updated to prevent image clipping at the large screen sizes.
- background props not needed
- width prop not needed
- `objectFit` needs to be inside the GatsbyImage `imgStyle` prop so it's value can be accepted.
```suggestion
        <Box
          as={GatsbyImage}
          flex="1 1 50%"
          alignSelf="center"
          mt={{ base: 0, lg: 12 }}
          ml={{ base: 0, lg: 12 }}
          maxWidth={{ base: "560px", lg: "624px" }}
          image={image}
          imgStyle={{
            objectFit: "contain",
          }}
          alt={alt}
          loading="eager"
        />
```

Reply:
Good call on using `imgStyle` TIL